django/db/migrations/operations/utils.py

```diff
@@ -14,6 +14,31 @@ def is_referenced_by_foreign_key(state, model_name_lower, field, field_name):
     return False
 
 
+def verbose_describe(operation, backwards):
```

Comment:
I'm not 100% convinced on the function name here. Maybe someone has a suggestion?
(Maybe it's fine.)